docs/src/layouts/MainLayout.vue

```diff
@@ -1,74 +1,80 @@
 <template>
   <q-layout view="hHh lpR fff" class="bg-lp-dark">
-
-    <q-header class="bg-lp-dark">
-      <q-toolbar>
+    <q-header class="bg-lp-dark horizontal-glow-bar">
+      <q-toolbar class="q-pa-md row justify-between">
         <q-btn flat @click="showDrawer = !showDrawer" round dense icon="menu" v-if="$q.screen.xs" color="lp-primary" />
-        <q-toolbar-title :class="$q.screen.xs? 'row justify-center items-center':''" class="add-vertical-bar position-relative">
-          <q-avatar>
-            <q-icon name="img:homepage-icons/quasar-logo-blue.svg" size="lg"/>
-          </q-avatar>
-          QUASAR
-        </q-toolbar-title>
-        <q-space v-if="$q.screen.gt.xs"/>
+        <div :class="$q.screen.xs? 'row justify-center items-center':''" class="add-vertical-bar position-relative">
+          <picture class="quasar-logo cursor-pointer" @click="$router.push({name: 'home'})">
+            <source srcset="https://github.com/quasarframework/quasar-art/blob/master/dist/svg/logo-dark.svg?raw=true" media="(min-width:810px) and (max-width:1080px)" width="48" height="48">
+            <img src="https://github.com/quasarframework/quasar-art/blob/master/dist/svg/logo-horizontal-dark.svg?raw=true" width="236" height="48" >
+          </picture>
+        </div>
 
-        <div class="row items-center">
+        <div class="row items-center text-size-16">
           <div v-if="$q.screen.gt.xs" class="toolbar-menu-items">
             <q-btn
```

Comment:
Would be cool to make this data-driven with a v-for being at this polishing stage, as all buttons have the same structure